src/Tribe/Assets.php

```diff
@@ -69,6 +69,18 @@ public function admin_enqueue_scripts() {
 			 */
 			$decimal = apply_filters( 'tribe_event_ticket_decimal_point', $decimal );
 
+			/**
+			 *  Filter Default Ticket Capacity Type
+			 *
+			 * @since TBD
+			 *
+			 * @param string 'global'
+			 *
+			 * @return string (global,own,unlimited)
+			 *
```

Comment:
not a blocker, but you can remove `@return` on hook docblocks

also I think `@param` should be more like `@param string $mode`

but again these aren't blockers so "fixing" is optional — tagging this both `merge` and `enhance` to reflect that

Reply:
I'll second the docblock comments and second that they're non-blockers 😉